app/controllers/service_center_capacities_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class ServiceCenterCapacitiesController < ApplicationController
+  def new
+    @set_count = ServiceCenterCapacity.new
+  end
+
+  def create
+    @set_count = ServiceCenterCapacity.new(allowed_parameter)
+    if @set_count.save
+      flash[:status] = 'count set successfully'
+      redirect_to service_center_admins_path
+    else
+      flash[:status] = 'count not set successfully'
```

Comment:
please render partial in else part .
what if the error occurs in save
it should be display in frontend side